vmm/src/memory_manager.rs

```diff
@@ -844,7 +835,7 @@ impl MemoryManager {
                 self.guest_ram_mappings.push(GuestRamMapping {
                     gpa: region.start_addr().raw_value(),
                     size: region.len(),
-                    slot,
+                    slot: 0,
```

Comment:
Could you please explain a bit more what you're doing here? Aren't you breaking the current behavior of Cloud Hypervisor?

Reply:
I am making 0 as the empty slot so that I know that I have to create a userspace mapping for this slot right before running the vCPU threads. I don't want to fall into the situation where we recreate userspace mappings for already allocated slots. What do you mean by breaking the current beahvior of Cloud Hypervisor?

Reply:
The first slot is 0 so how do you tell between the first slot or uninitialized slot. Use an `Option` type instead.